CDF Outreach Committee Charter.md

```diff
@@ -0,0 +1,115 @@
+
+# CDF Outreach Committee Charter
+
+
+## Mission Statement
+
+The Outreach Committee (OC) is the place where the community comes together, including the projects, the members, and the ambassadors with the goal to identify opportunities to help market and communicate Continuous Delivery Foundation's (CDF) vision, projects, Special Interest Groups (SIG) and Working Groups (WG) (new initiatives, new releases, need for contributors, etc.) within the CDF community and to the broader technical community in general. The OC Chair acts as the communication link between the board and the rest of the community.
```

Comment:
Should we link the place in the charter where Outreach Committee Chair is documented?

https://github.com/cdfoundation/charter/blob/master/CHARTER.md#4-outreach-committee